remix-tests/src/run.ts

```diff
@@ -44,8 +44,10 @@ commander
             log.info('verbosity level set to ' + commander.verbose.blue)
         }
         let web3 = new Web3()
+        let provider = new Provider()
```

Comment:
As you are not reassigning the web3 and provider object you can use `const`

Reply:
done